target/linux/brcm63xx/image/bcm63xx.mk

```diff
@@ -857,6 +857,20 @@ define Device/livebox
 endef
 TARGET_DEVICES += livebox
 
+### Netcomm ###
+define Device/NB604N
+  $(Device/bcm63xx)
+  IMAGES += sysupgrade.bin
+  DEVICE_TITLE := Netcomm NB604N
+  DEVICE_DTS := nb604n
+  CFE_BOARD_ID := 96328ang
+  CFE_CHIP_ID := 6328
+  FLASH_MB := 8
+  DEVICE_PACKAGES := \
+    $(BRCMSMAC_PACKAGES) $(USB2_PACKAGES)
```

Comment:
This will match into one line.

Reply:
Could you please clarify what you mean by this?

Reply:
DEVICE_PACKAGES := $(BRCMSMAC_PACKAGES) $(USB2_PACKAGES)